source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/characteristics/adms/ADMSLimits.java

```diff
@@ -84,6 +84,24 @@
   public static final double BUILDING_LENGTH_MINIMUM = 0.001;
   public static final double BUILDING_LENGTH_MAXIMUM = 1_000;
 
+  public static final int ROAD_GRADIENT_MIN = -50;
+  public static final int ROAD_GRADIENT_MAX = 50;
+  public static final int ROAD_ELEVATION_MIN = 0;
+  public static final int ROAD_ELEVATION_MAX = 15_000;
+  public static final int ROAD_WIDTH_MIN = 2;
+  public static final int ROAD_WIDTH_MAX = 100;
+  public static final int ROAD_COVERAGE_MIN = 0;
+  public static final int ROAD_COVERAGE_MAX = 100;
+
+  public static final double ROAD_POROSITY_DEFAULT_NONE = 0D;
+  public static final double ROAD_POROSITY_DEFAULT_NOISE_BARRIER = 5D;
+  public static final double ROAD_POROSITY_DEFAULT_BRICK_WALL = 5D;
+  public static final double ROAD_POROSITY_DEFAULT_STREET_CANYON_TERRACED_HOUSES = 20D;
+  public static final double ROAD_POROSITY_DEFAULT_STREET_CANYON_SEMIDETACHED_HOUSES = 40D;
+  public static final double ROAD_POROSITY_DEFAULT_STREET_CANYON_DETACHED_HOUSES = 50D;
+  public static final double ROAD_POROSITY_DEFAULT_TREE_BARRIER_DENSE = 80D;
+  public static final double ROAD_POROSITY_DEFAULT_TREE_BARRIER_OPEN = 95D;
```

Comment:
If these defaults are not bound to be changed anytime soon this is OK for me. But perhaps they could be defined in the enum instead so you don't have to do some switch statement or something when using these?

Reply:
Check!